skabase/SKALogger/test/SKALogger_test.py

```diff
@@ -85,8 +86,7 @@ def test_Status(self, tango_context):
     @pytest.mark.parametrize("logging_level", [int(tango.LogLevel.LOG_INFO)])
     @pytest.mark.parametrize("logging_target", ["logger/test/1"])
     # PROTECTED REGION END #    //  SKALogger.test_SetCentralLoggingLevel_decorators
-    def test_SetCentralLoggingLevel(self, tango_context, setup_log_test_device,
```

Comment:
`setup_log_test_device` might be unused in this function, but it is a test fixture and including it means that `SKATestDevice/SKATestDevice.py` will be launched.  Are we sure that is no longer required?

Reply:
We are instantiating separate container for SKATestDevice. However, for testing in the local environment, we still need this fixture. We had removed this in anticipation that we will do not more testing in the local environment. But as a second thought, we still may need to run test cases locally. So we will revert this change for some time and if not found useful, will remove it.